blogs/data-mover/data-mover-1.3/data-mover-1.3.0-overview.md

```diff
@@ -0,0 +1,230 @@
+     snapshot.</td>
+</tr>
+<tr>
+<td>dataMoverConfig</td>
+<td>DataMoverConfig is for data-mover-specific configuration fields.</td>
+</tr>
+<tr>
+<td>datamover</td>
+<td>DataMover specifies the data mover to be used by the backup. If DataMover
+     is "" or "velero", the built-in data mover will be used.</td>
+</tr>
+<tr>
+<td>operationTimeout</td>
+<td>OperationTimeout specifies the time used to wait internal operations,
+     before returning error as timeout.</td>
+</tr>
+<tr>
+<td>snapshotType</td>
+<td>SnapshotType is the type of the snapshot to be backed up. Currently the only valid value is CSI</td>
+</tr>
+<tr>
+<td>sourceNamespace</td>
+<td>SourceNamespace is the original namespace where the volume is backed up
+     from. It is the same namespace for SourcePVC and CSI namespaced objects.</td>
+</tr>
+<tr>
+<td>sourcePVC</td>
+<td>SourcePVC is the name of the PVC which the snapshot is taken for.</td>
+</tr>
+</tbody>
+</table>
+
+<h3>Note: For additional specification information please see the <a href=https://github.com/openshift/oadp-operator/blob/master/docs/API_ref.md>API reference documentation</a>
+
+<h3>DataUpload (DUCR) and DataDownload (DDCR) status descriptions</h3>
+
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>New</td>
+<td>The DUCR has been created but not processed by a controller</td>
+</tr>
+<tr>
+<td>Accepted</td>
+<td>The object lock has been acquired for this DUCR and the elected controller is trying to expose the snapshot</td>
+</tr>
+<tr>
+<td>Prepared</td>
+<td>The snapshot has been exposed, the related controller is starting to process the upload</td>
+</tr>
+<tr>
+<td>InProgress</td>
+<td>The data upload is in progress</td>
+</tr>
+<tr>
+<td>Canceling</td>
+<td>The data upload is being canceled</td>
+</tr>
+<tr>
+<td>Canceled</td>
+<td>The data upload has been canceled</td>
+</tr>
+<tr>
+<td>Completed</td>
+<td>The data upload has completed</td>
+</tr>
+<tr>
+<td>Failed</td>
+<td>The data upload has failed</td>
+</tr>
+</tbody>
+</table>
+
+<h2>Backup Process</h2>
+<div>
+A user creates a backup CR with the snapshotMoveData option set to true. Velero calls the BIA V2 api to create a CSI VolumeSnapshot request. The status will move from `New` to `InProgress`.<br><br>
+
+After the VolumeSnapshots are created, you will see one or more DataUpload CRs created.  You may also see some temporary objects (i.e., pods, PVCs, PVs) created in protected (openshift-adp) namespace. The temporary objects are created to assist in the data movement. The status of the DataUpload object will progress from `New` to `Accepted` to `InProgress`. <br>
+
+Now working from the Data Mover plugin the CSI snapshot is mounted from the Node-Agent. The DataUpload Controller then works with Kopia ( the uploader ) to move the object off cluster to the Unified Repo Backup repository off cluster.  The status is once again reconciled and the backup CR is moved to complete.<br>
```

Comment:
Maybe replace "Data Mover plugin" with "CSI plugin" as we are listing "CSI plugin" in the list of components involved.

Reply:
Now working from the CSI plugin the CSI snapshot is mounted from the Node-Agent.